addons/l10n_in_pos/static/src/overrides/components/pos_receipt.xml

```diff
@@ -26,6 +26,29 @@
                 </div>
             </t>
         </xpath>
+        <xpath expr="//div[hasclass('pos-receipt-taxes')]" position="replace">
```

Comment:
@maad-odoo replace is a bad idea. It can effect other localization, if you do it make sure it properly works with other localization too